modules/media/jsx/mediaIndex.js

```diff
@@ -163,7 +167,7 @@ class MediaIndex extends Component {
             <MediaUploadForm
               DataURL={`${loris.BaseURL}/media/ajax/FileUpload.php?action=getData`}
               action={`${loris.BaseURL}/media/ajax/FileUpload.php?action=upload`}
-              maxUploadSize={this.state.data.maxUploadSize}
+              maxUploadSize={this.state.fieldOptions.maxUploadSize}
```

Comment:
could this.state.fieldOptions just be the constant options defined above?

Reply:
sure could! I'll change that in a second